web/cypress/e2e/rest/service-edit-service-with-upstream.cy.js

```diff
@@ -88,19 +87,16 @@ context('Edit Service with Upstream', () => {
     cy.contains('Search').click();
     cy.contains(data.serviceName).siblings().contains('Configure').click();
 
-    cy.wait(500);
-    cy.get(selector.nodes_0_host)
-      .click({
-        force: true,
-      })
-      .should('value', data.ip1);
-    cy.get(selector.input).should('be.disabled');
-
     cy.get(selector.upstreamSelector).click();
-    cy.contains('.ant-select-item-option-content', 'Custom').click();
-    cy.get(selector.nodes_0_host).should('not.be.disabled').clear().type(data.ip2);
-    cy.get(selector.nodes_0_port).type(data.port);
-    cy.get(selector.nodes_0_weight).type(data.weight);
+    cy.wait(500);
+    cy.contains('.ant-select-item-option-content', 'Custom').should('be.visible').click();
+    cy.wait(300);
+    cy.get$(selector.nodes_0_host).should('not.be.disabled').clear().type(data.ip2);
```

Comment:
I use jQuery to wrap some elements, which can make the positioning of more accurate